tools/manifest-generator/manifest-generator.go

```diff
@@ -190,10 +216,9 @@ func generateFromFile(templFile string) {
 	}
 	defer file.Close()
 
-	if strings.Contains(*csvVersion, initialCsvVersion) {
-		data.ReplacesCsvVersion = ""
-	} else {
-		klog.Fatalf("Need to implement CSV upgrade to set ReplacesVersion in CSV when CSVVersion is greater than %s\n", initialCsvVersion)
+	data.ReplacesCsvVersion, err = evalOlmCsvUpdateVersion(templFile, *csvVersion, *bundleOut, *quayNamespace, *quayRepository)
+	if err != nil {
+		klog.Fatalf("Failed to evaluate CSV Replaces Version!", csvVersion)
```

Comment:
2 things.
1. I don't see a parameter to display csvVersion
2. Can you also display the error with a %v or something, so it easier to figure out exactly what went wrong.

Reply:
Done